features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -1430,6 +1430,24 @@
 				<SupportedByDefault />
 				<MappedLocalClaim>http://wso2.org/claims/url</MappedLocalClaim>
 			</Claim>
+			<Claim>
+				<ClaimURI>upn</ClaimURI>
+				<DisplayName>User Principal</DisplayName>
+				<AttributeID>upn</AttributeID>
```

Comment:
are these attributes available with the embedded LDAP by default?

Reply:
@pulasthi7 Fixed with 1c6ddc3ba697f347c87ab31fc9bae90508fc3644 as @emswbandara suggested.